openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -444,7 +447,9 @@ Note: Applications MAY combine authorization details of type `openid_credential`
 
 In addition to a mechanism defined in (#credential-authz-request), Credential Issuers MAY support requesting authorization to issue a credential using OAuth 2.0 scope parameter.
 
-When the Wallet does not know which scope value to use to request issuance of a certain credential, it can discover it using the `scope` Credential Issuer metadata parameter defined in (#credential-metadata-object). When the flow starts with a Credential Offer, the Wallet can use the information in the `credentials` Credential Offer parameter. When the entry of the `credentials` parameter is a string, it can be used as a scope value. When the entry of the `credentials` parameter is an object, it can be used to discover the desired scope value from the Credential Issuer metadata that can be obtained using `credential_issuer` Credential Offer parameter as defined in (#credential-issuer-wellknown).
+When the Wallet does not know which scope value to use to request issuance of a certain credential, it can discover it using the `scope` Credential Issuer metadata parameter defined in (#credential-metadata-object). When the flow starts with a Credential Offer, the Wallet can use the information in it as following:
+  * When the entry of the `credentials` Credential Offer parameter is a string, it can be directly used as a scope value.
+  * When the entry of the `credentials` Credential Offer parameter is an object, it can be used to discover the desired `scope` value from the Credential Issuer metadata that can be obtained using `credential_issuer` Credential Offer parameter as defined in (#credential-issuer-wellknown).
```

Comment:
agreed that a key is needed when doing a look up in `credentials_supported` entry that matches `credentials` entry. also based on your comment above, I think an optional `identifier` in the offer might be better suited for this purpose, because the scope value will not always be unique to a credential being offered.

Reply:
> an issuer may want to offer credentials that don't appear in the metadata, namely because they are very context specific.

How does the wallet know that it is being offered such a credential? I think this requires a separate issue.

Reply:
this topic is out of scope for this PR, I will revert this change. I was just trying to clarify the text, so my changes here did not introduce any normative changes.